cluster/gce/configure-vm.sh

```diff
@@ -287,6 +287,7 @@ manifest_url: '$(echo "$MANIFEST_URL" | sed -e "s/'/''/g")'
 manifest_url_header: '$(echo "$MANIFEST_URL_HEADER" | sed -e "s/'/''/g")'
 num_nodes: $(echo "${NUM_NODES}")
 e2e_storage_test_environment: '$(echo "$E2E_STORAGE_TEST_ENVIRONMENT" | sed -e "s/'/''/g")'
+kubelet_port: '$(echo "$KUBELET_PORT")'
```

Comment:
Not all lines are passed though the sed, e.g. two lines above there's a NUM_NODES variable which is not. I can add sed to all lines that are missing it, if you think it's needed.